### ISSUE TITLE
Reduce cognitive load of large number of shared docs

### ISSUE THREAD

**Issue by dvanic** [dvanic/metametalesson#12]

**Comment by ragamouf** [comment]
As a learner in the pilot, the impact on having a large number of shared docs at the start was that I assumed I needed to be across all of the docs, and my objective was to learn everything you're outlining. 

On reflection about attempting the first group activity, i realised I was shooting for getting the group to have 2 x draft personas started, that is, complete the task in front which asked for 2. I could have better spent that time getting us all to collaborate on just 1 persona. My problem became how do we create 2 generic personas in 5-10 minutes according to the terms of the activity. 
Alternative activities could be

- starting to build a concept map of the kind of persona I wanted to develop
- choosing the most appropriate persona from a list of likely ones.

One way to reduce the cognitive load of all the resources is to centre the learners' first activities on getting to know each other in the space and talking about the kind of learners they have in mind that they intend to be instructing. I think that if I had started the workshop with a clear idea of the kind of persona i wanted to develop and the lesson that I wanted to deliver to them, i would have gone into a persona development activity more sure that the outcome was more closely aligned to why i was attending the workshop.

**Comment by dvanic** [comment]
Thanks for the detailed feedback!

>  i realised I was shooting for getting the group to have 2 x draft personas started, that is, complete the task in front which asked for 2. I could have better spent that time getting us all to collaborate on just 1 persona. My problem became how do we create 2 generic personas in 5-10 minutes according to the terms of the activity.

The problem was actually that while we told you about formative assessments in part 2, we actually hadn't developed any for part 1 ! (or 2) - so you were thrown in the deep end without scaffolding - and, yes, time limits were why, but this will be a 3 hour workshop with no more material added apart from exercises.

> starting to build a concept map of the kind of persona I wanted to develop

Concept maps are both hard and actually -at least for the moment - outside the scope of the course.

> choosing the most appropriate persona from a list of likely ones.

Yes as a scaffolding/formative assessment. But to get over the hump of the blank page you really did need to try to write one yourselves - and you did start getting there as a group, at about t=10 min.

>  think that if I had started the workshop with a clear idea of the kind of persona i wanted to develop and the lesson that I wanted to deliver to them, i would have gone into a persona development activity more sure that the outcome was more closely aligned to why i was attending the workshop.

On reflection, we thought that we should have given you more of a brief of the workshop you were developing, and the summative assessment, to better keep things on track/keep it clear.

**Comment by ragamouf** [comment]
Yes - this workshop doesn't need more activities. Concept maps need a lot to unpack around them. 

I agree that more time briefing on the activity and summative assessment will help learners to be at less risk of getting lost.

Also - I had missed the first 5 minutes, so you may have gently lowered the eggs into the pan in your initial introduction but having missed that I was more focussed on catching up on lost time rather than really asking myself if I understood the brief. Thank you! This is really getting me to think hard about meta instruction.

**Comment by dvanic** [comment]
I think the other important caveat to mention: you are already an instructor, Liz, and have gone through the training to be a trainer. So you're actually coming in with a LOT more background/knowledge than some of our expected attendees of the training (you're still within the persona "range", but kind of at the top end...).
